client/src/contexts/Okteto.context.tsx

```diff
@@ -1,43 +1,54 @@
 import { ExecProcess } from '@docker/extension-api-client-types/dist/v1';
-import { createContext, useContext, useState, ReactNode, useEffect } from 'react';
+import {
+  createContext,
+  useContext,
+  useState,
+  ReactNode,
+  useEffect,
+} from 'react';
 import useInterval from 'use-interval';
 
 import okteto, { OktetoContext, OktetoContextList } from '../api/okteto';
 
 interface OktetoEnvironment {
-  file: string
-  link: string
-  contextName: string
-  process: ExecProcess | undefined
+  file: string;
+  link: string;
+  contextName: string;
+  process: ExecProcess | undefined;
 }
 
 interface OktetoStore {
-  currentContext: OktetoContext | null
-  contextList: OktetoContextList
-  environment: OktetoEnvironment | null
-  output: string
-  loading: boolean
-  ready: boolean
-
-  login: () => void
-  stopEnvironment: () => void,
-  selectEnvironment: (f: string, withBuild: boolean) => void
-  selectContext: (f: string) => void
+  currentContext: OktetoContext | null;
```

Comment:
It seems you are using your own linter / prettier configuration. We are working on adding a specific linter config for this project, but until then can you send a PR with only the changes related to the issue?